specification/iotcentral/data-plane/readme.md

```diff
@@ -10,34 +10,37 @@ To see additional help and options, run:
 ---
 
 ## Configuration
+
+### Basic Information
+
 These are the global settings for the IoT Central API.
+
 ``` yaml
 openapi-type: data-plane
-tag: package-2019-10-28-preview
+tag: package-2021-04-30-preview
 ```
 
-### Tag: package-2019-10-28-preview
-These settings apply only when `--tag=package-2019-10-28-preview` is specified on the command line.
+### Tag: package-2021-04-30-preview
+These settings apply only when `--tag=package-2021-04-30-preview` is specified on the command line.
 
-```yaml $(package-2019-10-28-preview-global)
-input-file:
-  - Microsoft.IoTCentral/preview/2019-10-28-preview/iotcentralapps.json
-```
-```yaml $(package-2019-10-28-preview-app)
+```yaml $(tag) == 'package-2021-04-30-preview'
 input-file:
-  - Microsoft.IoTCentral/preview/2019-10-28-preview/iotcentral.json
+  - Microsoft.IoTCentral/preview/2021-04-30-preview/iotcentral.json
 ```
 
-```yaml
-batch:
-    - package-2019-10-28-preview-global : true
-    - package-2019-10-28-preview-app: true
+### Tag: package-1.0.0
```

Comment:
I'm confused about the 2021-04-30 (a date in the future which I'd expect this API version to be) and this other 1.0.0 version.

Reply:
@drwill-ms there are two updates in this PR. Moving our evergreen /preview branch forward from 2019-10-28 to 2020-04-30. This corresponds to the Swagger in the /preview folder. 

And then introducing our 1.0 GA version and hence the Swagger in the stable version. 

Our IoT Central APIs leverage path based versioning and not date based query parameter.

Reply:
If there is an existing preview version out there, that version should not go away. You can add another preview, but not remove one AFAIK.

I now understand the other versioning option, so I'm cool with that.